applications/CoSimulationApplication/co_sim_io/python/test_import.py

```diff
@@ -0,0 +1,38 @@
+import CoSimIO
+
+# print(CoSimIO.__str__)
+# help(CoSimIO)
+
+
+
+u = 15
+
+def func_print_u():
+    print(u)
+
+
+# CoSimIO.FunctionCapture(func_print_u)
+
+def ProceedToNextTimeStep(new_time):
+
+
+    return new_time + 0.1
+
+def SolveSolutionStep():
+    print("SolveSolutionStep!", u)
+
+def SolveSolutionStep2(some_var):
+    print("SolveSolutionStep!")
+
+connection_name = "Python_solver"
+
+CoSimIO.Connect(connection_name, "py_solver_settings.txt")
+
+# CoSimIO.FunctionCapture(SolveSolutionStep2)
+CoSimIO.Register_SolveSolutionStep(connection_name, SolveSolutionStep)
+
+
+# CoSimIO.Run(connection_name)
+
+
+CoSimIO.Disconnect(connection_name)
```

Comment:
@inigolcanalejo this is veryvery roughly how you would use it, I will push more in the next days.
The necessary changes in the backend are done, with this the things you need are really minimal
I will add a more elaborate example soon